Description: Disable binary stripping at build time
 Stripping is now done by dh_strip separately.
Author: James Cowgill <james410@cowgill.org.uk>
Forwarded: not-needed
Last-Update: 2014-06-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -276,9 +276,6 @@ else ()
     message("-- Shared Objects will be written into ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")        ## Optimize
-    if(CMAKE_COMPILER_IS_GNUCXX)
-        set(CMAKE_EXE_LINKER_FLAGS "-s")  ## Strip binary
-    endif(CMAKE_COMPILER_IS_GNUCXX)
 
     ## In release, add NDEBUG unless explicitly told not to
     if (NOT PREVENT_WX_ASSERTS MATCHES 0)
